tests/test-definition.yaml

```diff
@@ -5,7 +5,7 @@ dimensions:
       - "false"
   - name: spark
     values:
-      - 3.4.1
```

Comment:
3.4.1 is deprecated but still supported and therefore should be tested.

Reply:
readded 3.4.1, fixed example and ny jars https://github.com/stackabletech/spark-k8s-operator/pull/357/commits/83f1421c709ac70eb57382f01695a5b03d9a4bf3